cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
+    let customView = PaymentView()
+    let payment: Payment?
+
+    func callbacks() {
+        view.backgroundColor = UIColor(displayP3Red: 1, green: 1, blue: 1, alpha: 1)
+        customView.didTapButton = {
+            if let payment = self.payment {
+                self.delegate.didFinishFlow(amount: payment.amount
+                                            currency: payment.currency)
+            }
+        }
+    }
```

Comment:
the function should have a single responsibility 
changing background and reporting an action could be divided into 2 methods to avoid running unneeded code while scaling the view controller